pkg/synchromanager/clustersynchro_manager.go

```diff
@@ -430,6 +430,9 @@ func (manager *Manager) UpdateClusterValidatedCondition(name string, synchro *cl
 
 func (manager *Manager) UpdateClusterStatus(ctx context.Context, name string, status *clusterv1alpha2.ClusterStatus) error {
 	return manager.updateClusterStatus(ctx, name, func(clusterStatus *clusterv1alpha2.ClusterStatus) {
+		if status.APIServer != "" {
+			clusterStatus.APIServer = status.APIServer
+		}
```

Comment:
`UpdateClusterValidatedCondition` is the first place to set the status, and it looks like status.apiserver needs to be set as well, otherwise if synchro fails to create, status.apiserver will always be empty.

Reply:
Is it ok to update `apiserver` in  `UpdateClusterValidatedCondition` ? 
`UpdateClusterValidatedCondition` seems  to be  be responsible for updating status condition.
And the `version`   also has not updated in `UpdateClusterValidatedCondition`.

Reply:
version is actually a runtime state (determined by the cluster), so it is better to set it in cluster synchro, we can also check the version once before creating cluster synchro but it is not necessary and will cause the apiserver of the member to be accessed every time it is reconciled.

The address of the apiserver is a static information that will not change as long as the apiserver or kubeconfig is not modified, and we can just set it in `UpdateClusterValidatedCondition`.

However, this also makes the semantics of the `UpdateClusterValidatedCondition` less accurate, so we can change the function name to `UpdateClusterAPIServerAndValidatedCondition` or something else.

Reply:
> The version is actually a runtime state (determined by the cluster), so it is better to set it in cluster synchro, we can also check the version once before creating cluster synchro but it is not necessary and will cause the apiserver of the member to be accessed every time it is reconciled.
> 
> The address of the apiserver is a static information that will not change as long as the apiserver or kubeconfig is not modified, and we can just set it in `UpdateClusterValidatedCondition`.
> 
> However, this also makes the semantics of the `UpdateClusterValidatedCondition` less accurate, so we can change the function name to `UpdateClusterAPIServerAndValidatedCondition` or something else.

OK， Do you think we should reserve the apiserver-updating in cluster synchro

Reply:
> OK， Do you think we should reserve the apiserver-updating in cluster synchro

It doesn't change in the cluster synchro, so I think it's okay not to update the apiserver.